src/blog/on-removing-let-let-mut.vmd

```diff
@@ -0,0 +1,330 @@
+---
+title: On Removing Let and Let Mut
+author: Evan Ovadia
+date: Apr 17, 2022
+realm: blog
+path: blog/on-removing-let-let-mut
+layout: annotated
+namespace: c-blog m-annotated
+sponsor: us
+---
+
+We're going to commit a cardinal sin today and talk about *syntax design!* [# Normally we talk about new memory management schemes, yak shaving contests, and other nonsense. It feels weird to be talking about something other people usually talk about!]
+
+
+<div><img src="/images/bikeshed.jpg" width="100%" style="opacity: .8"/><center><span style="opacity: .8"><a href="https://en.wikipedia.org/wiki/Law_of_triviality">(Also, while we have you here, what color should we paint this shed?)</a></span></center></div>
+
+
+We often think that there are only three alternatives for variable declaration:
+
+ * One keyword to declare a variable, such as:
+    * In C++, `auto x = 4;`
+    * Swift's `var x = 4`
+    * Go's `x := 4` (though it's not really a keyword per se)
+ * Specifying whether the variable can change, such as:
+    * Rust's `let x = 4;` vs `let mut x = 4;`
+    * Javascript's `let x = 4` vs `const x = 4`
+ * Leaving it off completely, such as:
+    * Python's `x = 4`
+
+
+The previous version of [Vale](https://vale.dev/) used the second option, `let` and `let mut`. [# Vale is a language that aims to be fast and memory-safe, while still being easy to learn. This syntax change helps with that!]
+
+However, it recently changed to a fourth option, which was so nice that it became the default behavior in version 0.2.
+
+
+<slice/>
+
+
+# The Fourth Option
+
+Most languages can't just use `x = 4;`, because that already meant something; that's the *re-assignment statement*, which modifies an existing variable that we already declared.
```

Comment:
Refactor this sentence, likely into two because its fairly run on. Also as a general rule, do not use `;`, its the footgun of english grammar. Using it properly is very rare